core/templates/dev/head/pages/exploration_editor/exploration_objective_editor_directive.html

```diff
@@ -0,0 +1,24 @@
+<form name="objectiveForm">
+  <p ng-class="{'has-error': objectiveForm.$dirty && (!explorationObjectiveService.displayed || explorationObjectiveService.displayed.length < 15 || explorationObjectiveService.displayed.length > 100)}">
+    <label for="explorationObjective" ng-style="<[::formStyle]>"><[::labelText]></label>
+    <input id="explorationObjective" type="text"
+          class="form-control protractor-test-exploration-objective-input"
+          ng-model="explorationObjectiveService.displayed"
+          ng-blur="onInputFieldBlur()" placeholder="Learn how to..." ng-trim="false">
+    <span class="help-block help-block-directive" ng-show="objectiveForm.$dirty && explorationObjectiveService.displayed.length < 15" style="font-size: smaller">
```

Comment:
Where does this "help-block-directive" class come from, and what does it do?

Reply:
I don't see where that CSS class is defined, though? Searching for "help-block-directive" on this page doesn't show me any other occurrences.

If the additional CSS needed is static, you can declare it here (or just add some stuff to the style attr on this tag). If it changes with context then you could probably pass in a custom ng-style through the directive, similar to what's done with formStyle.